public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <title>React Redux Template</title>
```

Comment:
- [ ] **[OPTIONAL]** It would be better to give your page a good title that describes your project. Maybe `Space Travelers Hub` or `Space-Xplore`. What do you think?